timescaledb/quick-start/java.md

```diff
@@ -0,0 +1,621 @@
+        }
+    }
+
+    private static void insertData(final Connection conn) throws SQLException {
+        final List<Sensor> sensors = List.of(
+                new Sensor("temperature", "bedroom"),
+                new Sensor("temperature", "living room"),
+                new Sensor("temperature", "outside"),
+                new Sensor("humidity", "kitchen"),
+                new Sensor("humidity", "outside"));
+        for (final var sensor : sensors) {
+            try (var stmt = conn.prepareStatement("INSERT INTO sensors (type, location) VALUES (?, ?)")) {
+                stmt.setString(1, sensor.type());
+                stmt.setString(2, sensor.location());
+                stmt.executeUpdate();
+            }
+        }
+
+        final var sensorDataCount = 100;
+        final var insertBatchSize = 10;
+        try (var stmt = conn.prepareStatement("""
+                INSERT INTO sensor_data (time, sensor_id, value)
+                VALUES (
+                    generate_series(now() - INTERVAL '24 hours', now(), INTERVAL '5 minutes'),
+                    floor(random() * 4 + 1)::INTEGER,
+                    random()
+                )
+                """)) {
+            for (int i = 0; i < sensorDataCount; i++) {
+                stmt.addBatch();
+
+                if ((i > 0 && i % insertBatchSize == 0) || i == sensorDataCount - 1) {
+                    stmt.executeBatch();
+                }
+            }
+        }
+    }
+
+    private static void executeQueries(final Connection conn) throws SQLException {
+        try (var stmt = conn.prepareStatement("""
+                SELECT time_bucket('15 minutes', time) AS bucket, avg(value)
+                FROM sensor_data
+                JOIN sensors ON sensors.id = sensor_data.sensor_id
+                WHERE sensors.type = ? AND sensors.location = ?
+                GROUP BY bucket
+                ORDER BY bucket DESC
+                """)) {
+            stmt.setString(1, "temperature");
+            stmt.setString(2, "living room");
+
+            try (var rs = stmt.executeQuery()) {
+                while (rs.next()) {
+                    System.out.printf("%s: %f%n", rs.getTimestamp(1), rs.getDouble(2));
+                }
+            }
+        }
+    }
+
+    private record Sensor(String type, String location) {
+    }
+}
+```
+
+Congratulations 🎉, you've successfully executed a query on TimescaleDB using Java and PostgreSQL JDBC!
+
+## Next steps
+
+Now that you're able to connect, read, and write to a TimescaleDB instance from your Java application, 
+be sure to check out these advanced tutorials:
+
+* Get up and running with TimescaleDB with our [Getting Started][timescaledb-getting-started] tutorial.
+* Refer to the [PostgreSQL JDBC Driver documentation][pg-jdbc-driver-docs] for more information.
+
+[timescaledb-install]: /how-to-guides/install-timescaledb/
+[timescaledb-getting-started]: /getting-started/
+[timescaledb-hypertable]: /overview/core-concepts/
```

Comment:
```suggestion
[timescaledb-hypertable]: /overview/core-concepts/hypertables-and-chunks/
```

Reply:
Just correct this one link but otherwise great! You've done a great job adapting what we had. As @Loquacity mentioned, it's obvious that all of these Quick Starts are in need of some attention to build further.  This is a good reminder to do that!

Thanks again for the contribution!!